### PR TITLE
Clear button for calculator

### PR DESCRIPTION
**Pull request by sirrus233**
Does what it says - zeroes out all the unit counts

image

### CODE REVIEW

File: frontend/src/ProductionHelper.tsx [hunk 1]
```diff
@@ -62,17 +68,28 @@ export default function ProductionHelper() {
...
+                        imgPath={getImgPath(unit)}
+                        onChange={handleUnitCountChange}
+                    />
+                ))}
+            </Stack>
+
+            <Box>
+                <Button
+                    color={"danger"}
+                    onClick={() => handleClearUnitCounts()}
```
Comment (chelseaSchmidt) [inline frontend/src/ProductionHelper.tsx]: Nit: `onClick={handleClearUnitCounts}`. Nbd if you'd rather just merge though!

**Review by chelseaSchmidt — APPROVED** [review]
Great idea! This is a nice touch ✨

File: frontend/src/ProductionHelper.tsx (hunk 1 above)
Comment (sirrus233) [reply frontend/src/ProductionHelper.tsx]: What, you don't think I should've went the other way and gone with `() => (() => handleClearUnitCounts())()` ??

Comment (chelseaSchmidt) [reply frontend/src/ProductionHelper.tsx]: 😂

**Pull request merged**